### PR TITLE
IMS-multi-CF-deployment

### PR DESCRIPTION
**Pull request by apinnick**
Document deployment of multiple CF appliances for IMS 1.2+

Doc preview: http://file.tlv.redhat.com/~apinnick/ims-multi-cf-deployment_IMS_1.2/

### CODE REVIEW

**Comment by apinnick** [comment]
On QE

File: doc-Infrastructure_Migration_Solution_Guide/cfme/modules/proc_Installing_configuring_multi-cf.adoc [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+[NOTE]
+====
+If you create an infrastructure mapping or migration plan and then remove or change a provider in CloudForms, the mapping or plan must be deleted and re-created.
+====
+
+.Procedure
+
+ifdef::rhv_1-2_vddk,rhv_1-3_vddk[]
+. Install Red Hat CloudForms on link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html/installing_red_hat_cloudforms_on_red_hat_virtualization[Red Hat Virtualization (RHV)] or link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html/installing_red_hat_cloudforms_on_vmware_vsphere/index[VMware].
```
Comment (mnadeem92) [inline doc-Infrastructure_Migration_Solution_Guide/cfme/modules/proc_Installing_configuring_multi-cf.adoc:18]: Query: Is the installation of CFME appliance is dependent on provider based on destination provider used in IMS as mentioned in guide
for vSphere->RHV = CFME should install on vSphere or RHV only
for vSphere->OSP = CFME should install on vSphere or OSP only

File: doc-Infrastructure_Migration_Solution_Guide/cfme/modules/proc_Installing_configuring_multi-cf.adoc [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+. link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html-single/general_configuration/index#servers[Enable the required server roles] in the CloudForms user interface:
+
+* Database: No server roles required
+* User interface:
+
+**  Database Operations
+**  User Interface
+**  Web Services
+**  Websocket
```
Comment (mnadeem92) [inline doc-Infrastructure_Migration_Solution_Guide/cfme/modules/proc_Installing_configuring_multi-cf.adoc:32]: I could not find any "Websocket" option in server roles, I have checked it in  CFME version 5.11.4.2 and also in upstream appliance ManageIQ Kasparov, Could you please have a look into it, I have attached screenshot for the same.

File: doc-Infrastructure_Migration_Solution_Guide/cfme/modules/proc_Installing_configuring_multi-cf.adoc [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+ifdef::rhv_1-2_vddk,rhv_1-3_vddk[]
+. Install Red Hat CloudForms on link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html/installing_red_hat_cloudforms_on_red_hat_virtualization[Red Hat Virtualization (RHV)] or link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html/installing_red_hat_cloudforms_on_vmware_vsphere/index[VMware].
+endif::[]
+ifdef::osp_1-2_vddk,osp_1-3_vddk[]
+. Install Red Hat CloudForms on link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html-single/installing_red_hat_cloudforms_on_red_hat_openstack_platform/[Red Hat OpenStack Platform (RHOSP)] or link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html/installing_red_hat_cloudforms_on_vmware_vsphere/index[VMware].
+endif::[]
+
+. link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html-single/general_configuration/index#servers[Enable the required server roles] in the CloudForms user interface:
+
+* Database: No server roles required
```
Comment (mnadeem92) [inline doc-Infrastructure_Migration_Solution_Guide/cfme/modules/proc_Installing_configuring_multi-cf.adoc:26]: We are missing an important section here for configuring the worker appliance which is very much required to setup the multi-appliance environment, IMO please add this section as well.
Database : Should be a default appliance
User interface and Worker appliances needs to configured themselves as per below guide. 
>> https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html-single/installing_red_hat_cloudforms_on_red_hat_openstack_platform/index#configuring-a-worker-appliance

**Comment by mnadeem92** [comment]
cfme-server-roles

File: doc-Infrastructure_Migration_Solution_Guide/cfme/modules/proc_Installing_configuring_multi-cf.adoc (hunk 1 above)
Comment (apinnick) [reply doc-Infrastructure_Migration_Solution_Guide/cfme/modules/proc_Installing_configuring_multi-cf.adoc:18]: Yes, the ifdef tags indicate which guides have the instructions (RHV users see one thing, OSP another)

Please see the doc preview in the PR description.

File: doc-Infrastructure_Migration_Solution_Guide/cfme/modules/proc_Installing_configuring_multi-cf.adoc (hunk 3 above)
Comment (apinnick) [reply doc-Infrastructure_Migration_Solution_Guide/cfme/modules/proc_Installing_configuring_multi-cf.adoc:26]: Will do

File: doc-Infrastructure_Migration_Solution_Guide/cfme/modules/proc_Installing_configuring_multi-cf.adoc (hunk 2 above)
Comment (apinnick) [reply doc-Infrastructure_Migration_Solution_Guide/cfme/modules/proc_Installing_configuring_multi-cf.adoc:32]: You are correct. Although it's still in the documentation (https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html-single/general_configuration/index#region-aware-server-roles), I could not find it in the UI. Investigating

**Comment by mnadeem92** [comment]
+1 LGTM

**Comment by apinnick** [comment]
@RichardHoch  Do you want to review this?

**Comment by RichardHoch** [comment]
Everything looks fine. Approved.

**Pull request merged**